src/common/modules/HelperFunctions.js

```diff
@@ -10,3 +10,19 @@
 export function objectIsEmpty(obj) { // eslint-disable-line no-unused-vars
     return Object.keys(obj).length === 0 && obj.constructor === Object;
 }
+
+/**
+ * Try fn when the return promise is rejected
```

Comment:
`fn` abbreviations here. :smile: 
Also, if you are at it, also always add an empty line before the `@` params. This is how it's done in other JSDOC in this project, so just stay consistent here.

Reply:
i've change the description, sometimes im so bad at jsdocs.